server/apps/publish/subscribers.py

```diff
@@ -123,6 +132,30 @@ def on_create(self, docs):
     def on_update(self, updates, original):
         self._validate_seq_num_settings(updates)
 
+    def _get_subscribers_by_filter_condition(self, filter_condition):
+        req = ParsedRequest()
+        all_subscribers = list(super().get(req=req, lookup=None))
```

Comment:
True Peter. I had to have turn it to a list as I need to iterate and then re-iterate in the same function.